cpu/sensors.py

```diff
@@ -33,6 +33,8 @@ class Sensors(Test):
 
     """
     Run sensors command on ppc architectures.
+
+    :avocado: tags=cpu,power,privileged
```

Comment:
@harish-24 right, but it looks like *this* test is specific to power.  A few facts:
 * docstring says it: "Run sensors command on ppc architectures."
 * there's a specific check for `CONFIG_SENSORS_IBMPOWERNV` and the test is skipped if it's not set

Can you please double check your suggestion in light of those? Thanks.